src/main/java/fi/metatavu/famifarm/rest/V1RESTService.java

```diff
@@ -794,6 +855,36 @@ public Response listCampaigns() {
     return createOk(translatedCampaigns);
   }
 
+  @Override
+  @RolesAllowed({ Roles.ADMIN, Roles.MANAGER, Roles.WORKER })
+  public Response listCutPackings(Integer firstResult, Integer maxResults, UUID productId, String createdBeforeString, String createdAfterString) {
+    fi.metatavu.famifarm.persistence.model.Product productToFilterBy = null;
+
+    if (productId != null) {
+      fi.metatavu.famifarm.persistence.model.Product existingProduct = productController.findProduct(productId);
+
+      if (existingProduct == null) {
+        return createBadRequest("Product with id " + productId + " not found!");
```

Comment:
Use formatter instead of concat i.e. String.format("Product with id %s not found!", productId)